tests/analyze-fixtures/Statement/DoNotUseGlobals.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace Tests\Analyze\Fixtures\Statement;
+
+$a = 1;
+
+class DoNotUseGlobals {
+
+    /**
+     * test global
+     */
+    public function testGlobal()
+    {
+        global $a;
+    }
+
+    /**
+     * test no global
+     */
+    public function testNoGlobal()
+    {
+        $a = 2;
+
+        return $a * 3;
+    }
+
+}
+
+?>
+----------------------------
+[{
+    "type": "missing_docblock",
+    "message": "Missing Docblock",
+    "file": "DoNotUseGlobals.php",
+    "line": 6
+}, {
+    "type": "do_not_use_globals",
+    "message": "Do not use globals",
+    "file": "DoNotUseGlobals.php",
+    "line": 13
```

Comment:
you only need this one of type do_not_use_globals. you have to remove the other ones.